test/testiconv.c

```diff
@@ -115,10 +137,44 @@ int main(int argc, char *argv[])
         }
         test[0] = SDL_iconv_string("UTF-8", "UCS-4", ucs4, len);
         SDL_free(ucs4);
-        (void)fputs(test[0], stdout);
+        SDL_Log("%s", test[0]);
         SDL_free(test[0]);
     }
-    (void)fclose(file);
+    SDL_free(fdata);
+
+    #if 0
```

Comment:
?

Reply:
Was testing the StepUTF8 code, but it's not useful for automated testing; you have to manually inspect bytes in a text file.   :)

Reply:
Removed in 01198114b2567f060835604da54187f1002286c1.